pysrvx/srvx.py

```diff
@@ -141,7 +141,7 @@ def get_response(self):
                 raise NotConnected
 
             # Append the response buffer
-            self.response += tmp
+            self.response += tmp.decode('UTF-8')
```

Comment:
irc is not always utf8. need to fall back to latin1 or iso-8859-1 if utf8 decoding fails, and ascii ignore (loses anything that is not ascii) if that fails too